src/xrCore/xrDebug.cpp

```diff
@@ -67,6 +67,8 @@ static BOOL bException = FALSE;
 #define MACHINE_TYPE IMAGE_FILE_MACHINE_ARM64
 #elif defined XR_ARCHITECTURE_ARM
 #define MACHINE_TYPE IMAGE_FILE_MACHINE_ARM
+#elif defined XR_ARCHITECTURE_E2K
+#define MACHINE_TYPE IMAGE_FILE_MACHINE_E2K
```

Comment:
`MACHINE_TYPE IMAGE_FILE_MACHINE_*` is Windows-specific and it's not needed in our case. (and it's not even supported)